buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -35,6 +35,7 @@ object Versions {
     const val kakao = "2.3.0"
     const val flexBox = "2.0.1"
     const val browser = "1.2.0"
+    const val fakeit = "1.0.2"
```

Comment:
What's the purpose of the fakeit library? 

Also lets be aware of the library additions, cause we still have to enable R8 later on and it doesn't play well with the many dependencies

Reply:
The lib generates fake random data for our model instead of having Person 1, 2, ... Biography 1 ... and so on. However, we should remove it after implementing the repository properly.

Reply:
okay but a list of dummy data would have served the purpose too